pyproject.toml

```
@@ -6,17 +6,16 @@ authors = ["kolibril13 <44469195+kolibril13@users.noreply.github.com>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.8,<3.11"
-ipykernel = "^6.13.0"
-jupyterlab = "^3.3.4"
+python = "^3.8"
```

And again, scipy also sets the upper bound, so we need to further constrain as you did originally.

```suggestion
python = ">=3.8,<3.11"
```